### DISCUSSION TITLE
How to add numbering to bibliography

### DISCUSSION THREAD

**Discussion by boterocamilo** [typst/typst#1711] [Q&A]
Hi everyone,

I am new to this journey of using a different tool that is not a classical word processor when writing. Until now, I'm impressed with Typst and grateful to its developers.

Right now, I am facing a problem regarding the bibliography:

*1.* I citing a paper using _#cite("doc_name")_ :
<img width="481" alt="Screenshot 2023-07-13 at 10 45 17" src="https://github.com/typst/typst/assets/46091241/19841047-94f5-4632-a851-c36da955d5a3">

*2.* This is the result I have:
<img width="470" alt="Screenshot 2023-07-13 at 10 45 44" src="https://github.com/typst/typst/assets/46091241/365b3ea8-3232-43e8-9ece-eb7a3f2754c3">
Here my first problem arises. It is inserting the APA citation instead of a number that redirects to the bibliography.

*3.* To generate the bibliography, I used the following:
<img width="401" alt="Screenshot 2023-07-13 at 10 46 29" src="https://github.com/typst/typst/assets/46091241/b5d52b3e-710a-4ed4-96ee-10d443e940b7">

*4.* The generated bibliography does not have the numbering I would like:
<img width="478" alt="Screenshot 2023-07-13 at 10 46 51" src="https://github.com/typst/typst/assets/46091241/52b25d25-de78-4c1b-82e5-989cb90765b5">

*5.* What I'm looking for it is something similar to the example it appears on Typst Documentation:
![elmqV5sqBUqdFbG6aR_oiQAAAAAAAAAA](https://github.com/typst/typst/assets/46091241/6bfae593-fb64-4aa2-a4d4-56a468a9eec8)

Does anyone know a solution for it? Or can point out where my error is?

Thanks in advance!

**Comment by laurmaedje** [comment]
If you change the bibliography's style to APA, the citations by default switch to an author date style. You can fix that by adding `#set cite(style: "numerical")` at the beginning of your document/template.
